packages/components/tooltip/src/tooltip.tsx

```diff
@@ -295,7 +296,29 @@ export const Tooltip = forwardRef<TooltipProps, "div">(
       () => (isOpen && closeOnScroll ? closeNow() : undefined),
     )
 
-    useEventListener(() => triggerRef.current, "pointerleave", closeWithDelay)
+    useEventListener(
+      () => triggerRef.current,
+      "pointerleave",
+      (e) => {
+        if (e.pointerType !== "touch") closeWithDelay()
+      },
+    )
+
+    useEventListener(
+      () => triggerRef.current,
+      "touchstart",
+      () => {
+        if (isOpen) closeWithDelay()
+        else openWithDelay()
+      },
+    )
```

Comment:
I want to handle toggle tooltip event on touchable device . So that's why i decide to add this one.